### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#7509] [Q&A] [answered]
Hello
From last night until now, Thermox has been facing an error for me
Please help me
My Android version is 10 and the name of the phone is Huawei Nova 5T
![Screenshot_20210911_053937_com termux](https://user-images.githubusercontent.com/90487160/132931931-043056e3-cec1-49d5-a54b-4fb70b97fa71.jpg)

**Comment by agnostic-apollo** [comment]
What did you understand or not understand from the error message?
Did you go to the `[4]` link and read?

**Comment by mkluth** [comment, marked as answer]
Termux no longer receiving updates from the Play Store (now deprecated). The error message telling you to migrate the app from the Play Store to the one released on F-Droid or the one released on Github. Thats it :)

**Comment by mkluth** [comment]
> Termux no longer receiving updates from the Play Store (now deprecated). The error message telling you to migrate the app from the Play Store to the one released on F-Droid or the one released on Github.  Thats it :)

Also, please make sure that you read and fully understand the error message given to you.

**Reply by ieyhan09** [reply]
![Uploading Screenshot_2021-09-11-12-59-31-37_66a3b1c2c181ff92b8bf945e748b6fb7.jpg…]()

**Comment by SDRausty** [comment]
https://f-droid.org/packages/com.termux/

https://search.f-droid.org/?q=Termux

**Comment by leap0x7b** [comment]
Thermox :kekw:
